http-server/src/main/java/org/triplea/modules/user/account/login/authorizer/anonymous/AnonymousLoginFactory.java

```diff
@@ -1,21 +1,6 @@
 package org.triplea.modules.user.account.login.authorizer.anonymous;
 
-import java.util.Optional;
-import java.util.function.Function;
 import lombok.experimental.UtilityClass;
-import org.jdbi.v3.core.Jdbi;
-import org.triplea.db.dao.UserJdbiDao;
-import org.triplea.domain.data.UserName;
-import org.triplea.modules.chat.Chatters;
 
 @UtilityClass
-public class AnonymousLoginFactory {
-
-  public static Function<UserName, Optional<String>> build(
-      final Jdbi jdbi, final Chatters chatters) {
-    return AnonymousLogin.builder()
-        .chatters(chatters)
-        .userJdbiDao(jdbi.onDemand(UserJdbiDao.class))
-        .build();
-  }
-}
+public class AnonymousLoginFactory {}
```

Comment:
I have the sneaking suspicion you forgot to remove this class 😉

Reply:
Twice even :grin: 
Was also missed earlier when we updated factory class pattern to be factory methods.